<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/gui/icons/yt_32.ico</normaloff>:/gui/icons/yt_32.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>446</width>
         <height>326</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>446</width>
         <height>326</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeView_sc">
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_file"/>
   <addaction name="action_append_from_file"/>
   <addaction name="action_save_gpx"/>
   <addaction name="action_export_fav"/>
   <addaction name="action_remove_internal_twins"/>
   <addaction name="action_remove_twins"/>
   <addaction name="actionConfigure"/>
   <addaction name="action_about_prog"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_open_file"/>
    <addaction name="action_append_from_file"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_export_fav"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about_prog"/>
    <addaction name="action_about_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_remove_twins"/>
    <addaction name="action_clone_point"/>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_export_fav">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/save_as_pg.png</normaloff>:/gui/icons/save_as_pg.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить в ПроГород</string>
   </property>
  </action>
  <action name="action_save_gpx">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/save_as_gpx.png</normaloff>:/gui/icons/save_as_gpx.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить в gpx</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/exit.png</normaloff>:/gui/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_check_all">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/check_all.png</normaloff>:/gui/icons/check_all.png</iconset>
   </property>
   <property name="text">
    <string>Отметить все</string>
   </property>
  </action>
  <action name="action_uncheck_all">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/uncheck_all.png</normaloff>:/gui/icons/uncheck_all.png</iconset>
   </property>
   <property name="text">
    <string>Сбросить выбор</string>
   </property>
  </action>
  <action name="action_append_from_file">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/open_add.png</normaloff>:/gui/icons/open_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить из файла...</string>
   </property>
   <property name="toolTip">
    <string>Добавить из файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/open.png</normaloff>:/gui/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_del_from_list">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/delete.png</normaloff>:/gui/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить из списка</string>
   </property>
  </action>
  <action name="action_about_prog">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/help.png</normaloff>:/gui/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="action_about_Qt">
   <property name="text">
    <string>О Qt...</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/save.png</normaloff>:/gui/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/save_as.png</normaloff>:/gui/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_set_icon">
   <property name="text">
    <string>Иконка...</string>
   </property>
   <property name="toolTip">
    <string>Установить иконку точки</string>
   </property>
  </action>
  <action name="action_clone_point">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/clone_pnt.png</normaloff>:/gui/icons/clone_pnt.png</iconset>
   </property>
   <property name="text">
    <string>Клонировать точку</string>
   </property>
  </action>
  <action name="action_remove_twins">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/duplicate_pg.png</normaloff>:/gui/icons/duplicate_pg.png</iconset>
   </property>
   <property name="text">
    <string>Удалить дубли с ПГ</string>
   </property>
   <property name="toolTip">
    <string>Удалить точки, похожие на точки из ПГ</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/config-icon.png</normaloff>:/gui/icons/config-icon.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_remove_internal_twins">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/gui/icons/twins.png</normaloff>:/gui/icons/twins.png</iconset>
   </property>
   <property name="text">
    <string>Удалить дубли</string>
   </property>
   <property name="toolTip">
    <string>Удалить похожие точки</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
